apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/MoreActions/NotesDialog.tsx

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { FormProvider, SubmitHandler, useForm } from "react-hook-form";
+import ChatBubbleBottomCenterIcon from "@heroicons/react/24/outline/ChatBubbleBottomCenterIcon";
+
+import { Button, Dialog } from "@gc-digital-talent/ui";
+import { Submit, TextArea } from "@gc-digital-talent/forms";
+import { Maybe, PoolCandidate } from "@gc-digital-talent/graphql";
+import { toast } from "@gc-digital-talent/toast";
+import { errorMessages, formMessages } from "@gc-digital-talent/i18n";
+
+import { useUpdatePoolCandidateStatusMutation } from "~/api/generated";
```

Comment:
This will also need to be updated to use the new `@gc-digital-talent/graphql` package stuff.

Reply:
Fixed by [99507d7](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8853/commits/99507d7d39cbf934cfe3208692fc188e90e7f0b3)